packages/core/src/overlay.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+
+let portalNode = React.createRef<HTMLDivElement>()
```

Comment:
can now be part of `OverlayContextProvider`, right?

Reply:
I need it both in `OverlayContextProvider` and in the initialization of the Context. Using `createRef` twice there didn't work for me, so I'll leave it like this for now.